laanwj,2018-08-11T22:23:25Z,utACK 5654efb187d24eb29a343c720e3937b01457c8b7,https://github.com/bitcoin/bitcoin/pull/13944#issuecomment-412305195,412305195,
promag,2018-08-12T00:04:03Z,utACK 5654efb.,https://github.com/bitcoin/bitcoin/pull/13944#issuecomment-412309325,412309325,
donaloconnor,2018-08-12T09:20:27Z,utACK https://github.com/bitcoin/bitcoin/commit/5654efb187d24eb29a343c720e3937b01457c8b7,https://github.com/bitcoin/bitcoin/pull/13944#issuecomment-412329860,412329860,
fanquake,2018-08-12T09:45:38Z,utACK 5654efb,https://github.com/bitcoin/bitcoin/pull/13944#issuecomment-412330990,412330990,
MarcoFalke,2018-08-11T21:54:41Z,"All of our arguments are options, so our name of this method can be left untouched.",https://github.com/bitcoin/bitcoin/pull/13944#discussion_r209438389,209438389,test/functional/example_test.py
domob1812,2018-08-12T09:34:28Z,I assume that no existing code relies on `self.args` (as otherwise you would see test failures)?,https://github.com/bitcoin/bitcoin/pull/13944#discussion_r209449177,209449177,test/functional/test_framework/test_framework.py
laanwj,2018-08-12T09:43:50Z,"indeed, none of the code uses that\nas @MarcoFalke mentioned above, all the command-line arguments are options",https://github.com/bitcoin/bitcoin/pull/13944#discussion_r209449354,209449354,test/functional/test_framework/test_framework.py
